LogicLayer/Notifications/IMailSender.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace GameBoard.LogicLayer.Notifications
+{
+    public interface IMailSender
+    {
+        Task SendEmailConfirmationAsync(string email, string link);
+        Task SendEventCancellationAsync(IEnumerable<string> emails, string link);
+        Task SendEventConfirmationAsync(IEnumerable<string> emails, string link);
```

Comment:
What is `EventConfirmation`? "An event you are taking part in has been confirmed." is what the html says, but I don't understand it.

Reply:
We had a system in mind, in which after the cretor decides all event details, he confirms the event - this event is supposed to be sent when the even has been confirmed this way. I know it comes from an old big model that we won't have time to complete, but it's there. I can remove it if you want to.

Reply:
I don't really remember this model, but it surely isn't included in any forseeable US, so you should remove it.

Reply:
Done